dev/archery/archery/benchmark/google.py

```diff
@@ -78,32 +77,25 @@ class GoogleBenchmarkObservation:
     RegressionSumKernel/32768/0_mean            1 us          1 us  25.6307GB/s
     RegressionSumKernel/32768/0_median          1 us          1 us  25.7066GB/s
     RegressionSumKernel/32768/0_stddev          0 us          0 us  288.046MB/s
-
-    As from benchmark v1.4.1 (2019-04-24), the only way to differentiate an
-    actual run from the aggregates, is to match on the benchmark name. The
-    aggregates will be appended with `_$agg_name`.
-
-    This class encapsulate the logic to separate runs from aggregate . This is
-    hopefully avoided in benchmark's master version with a separate json
-    attribute.
```

Comment:
I manually tested this change with the following command and compared the Archery output before & after.

`$ archery benchmark run --suite-filter=arrow-compute-vector-selection-benchmark --benchmark-filter=TakeStringRandomIndicesWithNulls --repetitions=3`